.python-version

```diff
@@ -0,0 +1 @@
+3.12.3
```

Comment:
Is this file from pyenv? Is it needed or should it be on `.gitignore`?

Reply:
it is for pyenv and for the GitHub action that automatically sets the python version based on this file

Reply:
Fair enough, thanks for the clarification